src/form-update.rst

```diff
@@ -98,6 +98,12 @@ These changes can be made as:
 
   "Example_form", "example_id", "English", "2017120701"
 
+.. note::
+
+  You need to use the same filename when generating newer versions of your form since the XLSForm generator pays attention to the filename. If you use different filenames, you would need to delete the previous version to upload the newer version of form.
+
+  In the examples below, we have different filenames since that is necessary on the server.
```

Comment:
It's not clear what the "server" is here. I think you mean the Docs server, right?

So "In the examples below, we use unique filenames since unique filenames are required to upload them to this documentation website". Or something like that.

Reply:
Yes, that is right. I will fix the language :sweat_smile: